src/keypoint_pairs.py

```diff
@@ -0,0 +1,82 @@
+mediapipe_keypoint_pairs = [
```

Comment:
A good idea and common practice in python is to write global constants in all caps, like `MEDIAPIPE_KEYPOINT_PAIRS`. Therefore everyone looking at the code know that this is a constant and not a function or something else.